src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -275,6 +275,23 @@ class PostTyper extends MacroTransform with IdentityDenotTransformer  { thisTran
               case tpe => tpe
             }
           )
+        case Import(expr, selectors) =>
+          val exprTpe = expr.tpe
+          def checkIdent(ident: Ident): Unit = {
+            val name = ident.name.asTermName.encode
+            def isDottyLanguageFeature = { // TODO Add new language features to scala.language
```

Comment:
Why make an exception for language features? I would doubt we need one, but if we do, this needs to be explained in a comment.